x-pack/plugins/discover_enhanced/ui_tests/README.md

```diff
@@ -0,0 +1,17 @@
+## How to run tests
+First start the servers with
+
+```bash
+// ESS
+node scripts/playwright_start_servers.js --stateful
+// Serverless
+node scripts/playwright_start_servers.js --serverless=es
+```
+
+Then you can run the tests multiple times in another terminal with:
```

Comment:
It does means that at this point all tests will always run against all the servers we have started right? Are we still going to implement what we have been discussing of a given test to be able to declare it only wants to run in a specific environment through its config?

Reply:
We will use tagging to explicitly set if test is compatible for stateful, serverless or both deployment types. I think we can add it in the follow-up PR.